spec/models/document_spec.rb

```diff
@@ -246,4 +246,20 @@
       end
     end
   end
+
+  describe ".referenced_in_decision_notice" do
+    let(:document) do
+      create(:document, referenced_in_decision_notice: true, numbers: "REF1")
+    end
+
+    before { create(:document, referenced_in_decision_notice: false) }
+
+    it "returns documents where referenced_in_decision_notice is true" do
```

Comment:
Interesting. I test scopes explicitly which [David Chelimsky suggested (that's my posting, but I just copied him)](https://stackoverflow.com/a/60058513/1511504). However, you've implicitly tested both cases. Don't see why not 🤷‍♂️

Reply:
Hello, @notapatch, nice to have you back on the team!

I think I like the explicit approach...will amend!

Reply:
Thank you.